content/documentation.rst

```diff
@@ -0,0 +1,429 @@
+Documentation
+#############
+
+:menu_order: 007
+
+.. contents::
+
+
+Introduction
+============
+
+We have the following types of documentation:
+
+1. Code documentation
+2. Low level technical information
+3. Narrative documentation including User's Guides
+4. Marketing and promotional materials
+
+
+Narrative documentation
+=======================
+
+Overview
+--------
+
+Readers are mainly:
+
+* Administrators directly involved with SFTPlus. 
+  They can be in three stages - pre-sales, on trial, or existing customers.
+
+* Decision-makers involved in the implementation of the software. 
+  They can include the business and technical arm of the organization.
+
```

Comment:
the development team is also using the narrative documentation to check the "specifications" of a functionality.

the support team is using the narrative documentation to direct customers :)